src/mSupplyMobileApp.js

```diff
@@ -310,7 +312,8 @@ MSupplyMobileAppContainer.propTypes = {
   syncState: PropTypes.object.isRequired,
```

Comment:
I think when we are editing files we can slowly start to introduce default props while we're there rather than disabling, what do you think?

Reply:
I agree.